src/javascript/app/pages/cashier/payment_agent_list.js

```diff
@@ -1,6 +1,7 @@
 const BinaryPjax   = require('../../base/binary_pjax');
 const Client       = require('../../base/client');
 const BinarySocket = require('../../base/socket');
+const Url          = require('../../../_common/url');
 const urlForStatic = require('../../../_common/url').urlForStatic;
```

Comment:
if we have Url, we can remove urlForStatic and instead use `Url.UrlForStatic` in code below

Reply:
Yes, I fixed it now.